addons/website_theme_install/static/src/less/website_theme_install.less

```diff
@@ -1,42 +1,162 @@
+@o-theme-kanban-gray: #fcfcfc;
+
 .o_kanban_view.o_theme_kanban {
+    background-color: @o-theme-kanban-gray;
+
+    /// Un-grouped Layout (default)
     &.o_kanban_ungrouped {
-        .o-justify-content(center);
+        .o-justify-content(space-between);
+
+        @media (min-width: @screen-md-min ) {
```

Comment:
Just to annoy you: there is an useless extra space here... I'd remove it myself but as there are also other comments... ^^